src/ontology/cl-edit.owl

```diff
@@ -30820,6 +30834,119 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32555229") Annotation(ob
+AnnotationAssertion(rdfs:label obo:CL_4033022 "mucus secreting cell of bronchus submucosal gland")
+EquivalentClasses(obo:CL_4033022 ObjectIntersectionOf(obo:CL_0000319 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_8410043)))
+
+# Class: obo:CL_4033023 (airway submucosal gland collecting duct epithelial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25648514") Annotation(oboInOwl:hasDbXref "PMID:29656943") Annotation(oboInOwl:hasDbXref "PMID:31651362") obo:IAO_0000115 obo:CL_4033023 "An epithelial cell that is part of a collecting duct of an airway submucosal gland.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033023 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033023 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033023 "2023-03-16T15:20:28Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4033023 "airway submucosal gland collecting duct epithelial cell")
+SubClassOf(obo:CL_4033023 obo:CL_0000066)
+SubClassOf(obo:CL_4033023 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_8410077))
+
+# Class: obo:CL_4033024 (airway submucosal gland duct basal cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29656943") Annotation(oboInOwl:hasDbXref "PMID:31651362") Annotation(oboInOwl:hasDbXref "PMID:35046051") obo:IAO_0000115 obo:CL_4033024 "A basal cell that is part of a duct of an airway submucosal gland.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033024 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033024 "https://orcid.org/0000-0002-9185-3994")
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033024 "2023-03-16T15:51:39Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4033024 "airway submucosal gland duct basal cell")
+SubClassOf(obo:CL_4033024 obo:CL_0000646)
+SubClassOf(obo:CL_4033024 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_8410077))
+
+# Class: obo:CL_4033025 (perichondrial fibroblast)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "wikipedia:Perichondrium") obo:IAO_0000115 obo:CL_4033025 "A fibroblast that is part of the fibrous layer of the perichondrium. This cell is responsible for collagen fiber production.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033025 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033025 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033025 "2023-03-16T16:12:50Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4033025 "perichondrial fibroblast")
+EquivalentClasses(obo:CL_4033025 ObjectIntersectionOf(obo:CL_0000057 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0008305)))
+
+# Class: obo:CL_4033026 (lung perichondrial fibroblast)
```

Comment:
Unsure if there is a specific reason "lung" is used here and "airway" used in other terms. Ref describes "PC-fibro around the airway cartilage".

Reply:
If this cell type does exist in lung tissue, would you please add that reference to the term. The quote mentioned above refers to the airways as well.

Reply:
Looking at the [scRNA-seq data](https://5locationslung.cellgeni.sanger.ac.uk/all), the majority of cells identified as PC-fibro came from airway tissue, but a small proportion (~0'4%) was found in parenchymal tissue:
<img width="250" alt="image" src="https://user-images.githubusercontent.com/94959119/226394017-e2f96653-2df1-4c50-b95e-c867584cdc95.png">

Reply:
If that is from the dbxref, and if Par stands for parenchyma, it seems reasonable.